Source/TCPError.swift

```diff
@@ -30,17 +30,17 @@ public enum TCPError: ErrorProtocol {
     case operationTimedOut(description: String, data: Data)
     case closedSocket(description: String)
 
-    static func lastReceiveErrorWithData(source: Data, bytesProcessed: Int) -> TCPError {
+    static func lastReceiveError(withData source: Data, bytesProcessed: Int) -> TCPError {
```

Comment:
maybe this receiver error, send error is all wrong. what we mean is prefix or suffix of the data we're operating on. this could happen in other operations than send and receive.